src/widgets/DropdownButton.js

```diff
@@ -264,14 +264,19 @@ define(function (require, exports, module) {
             posTop = Math.max(0, toggleOffset.top - $dropdown.height() - 4);
         }
 
+        // Take in consideration the scrollbar to prevent the edit icon to move down. See #10963
+        var dropdownElement = this.$dropdown[0];
+        var scrollWidth = dropdownElement.offsetWidth - dropdownElement.clientWidth + 1;
```

Comment:
Also, where does the `+ 1` come from? Is it just a safety measure so we have a tiny little more space available or does it serve a real purpose?